src/test/scala/uk/gov/hmrc/hmrcfrontend/views/implicits/RichAccountMenuSpec.scala

```diff
@@ -17,64 +17,90 @@
         Table(
-          ("accountHomeActive", "messagesActive", "checkProgressActive", "yourProfileActive"),
-          (AccountHome(active = true), AccountMessages(), CheckProgress(), YourProfile()),
-          (AccountHome(), AccountMessages(active = true), CheckProgress(), YourProfile()),
-          (AccountHome(), AccountMessages(), CheckProgress(active = true), YourProfile()),
-          (AccountHome(), AccountMessages(), CheckProgress(), YourProfile(active = true))
+          (
+            "accountHomeActive",
+            "messagesActive",
+            "checkProgressActive",
+            "yourProfileActive",
+            "businessTaxAccountActive"
+          ),
+          (AccountHome(), AccountMessages(), CheckProgress(), YourProfile(), None),
+          (AccountHome(), AccountMessages(), CheckProgress(), YourProfile(), Some(BusinessTaxAccount())),
+          (AccountHome(active = true), AccountMessages(), CheckProgress(), YourProfile(), Some(BusinessTaxAccount())),
+          (AccountHome(), AccountMessages(active = true), CheckProgress(), YourProfile(), Some(BusinessTaxAccount())),
+          (AccountHome(), AccountMessages(), CheckProgress(active = true), YourProfile(), Some(BusinessTaxAccount())),
+          (AccountHome(), AccountMessages(), CheckProgress(), YourProfile(active = true), Some(BusinessTaxAccount())),
+          (AccountHome(), AccountMessages(), CheckProgress(), YourProfile(), Some(BusinessTaxAccount(active = true)))
         )
 
       forAll(activeLinkCombinations) {
         (
           accountHomeActive: AccountHome,
           messagesActive: AccountMessages,
           checkProgressActive: CheckProgress,
-          yourProfileActive: YourProfile
+          yourProfileActive: YourProfile,
+          businessTaxAccountActive: Option[BusinessTaxAccount]
         ) =>
           val accountMenu: AccountMenu = AccountMenu(
             accountHome = accountHomeActive,
             messages = messagesActive,
             checkProgress = checkProgressActive,
-            yourProfile = yourProfileActive
+            yourProfile = yourProfileActive,
+            businessTaxAccount = businessTaxAccountActive
           ).withUrlsFromConfig()(enMessages, FakeRequest())
 
           accountMenu shouldBe AccountMenu(
             accountHome = AccountHome(href = defaultAccountHomeHref, accountHomeActive.active),
             messages = AccountMessages(href = defaultMessagesHref, messagesActive.active),
             checkProgress = CheckProgress(href = defaultCheckProgressHref, checkProgressActive.active),
             yourProfile = YourProfile(href = defaultYourProfileHomeHref, yourProfileActive.active),
+            businessTaxAccount = businessTaxAccountActive match {
+              case Some(businessAccount) =>
+                Some(BusinessTaxAccount(href = defaultBusinessTaxAccountHref, businessAccount.active))
+              case _                     => None
+            },
```

Comment:
small totally optional stylistic one
```suggestion
            businessTaxAccount = businessTaxAccountActive.map(_.copy(href = defaultBusinessTaxAccount)),
```